packages/integrations/useAxios/index.ts

```diff
@@ -42,12 +42,12 @@ export function useAxios<T = any>(url: string, ...args: any[]) {
       instance = args[1]
   }
 
-  const response = ref<any>(null) as Ref<AxiosResponse<T> | undefined>
-  const data = ref<any>(undefined) as Ref<T | undefined>
+  const response = shallowRef<any>(null) as Ref<AxiosResponse<T> | undefined>
```

Comment:
```suggestion
  const response = shallowRef<AxiosResponse<T> | null>(null)
```

Shouldn't we do this? Typed as undefined but passed as null feels a bit weird to me.

Reply:
Ah yeah haha, not sure what was up with the types there to begin with 😅